instagram-app/src/PostsContainer/ComentSection/CommentInput.js

```diff
@@ -0,0 +1,12 @@
+import React from "react";
+import "./CommentSection.css";
+
+const CommentInput = props => {
+  return (
+    <form onChange={props.handleCommentChange} onSubmit={props.handleSubmit}>
```

Comment:
You would want the onChange event listener to be on the input, but the entire form.